addons/fluentbit/1.5.x/fluentbit-4.yaml

```diff
@@ -102,19 +110,20 @@ spec:
               Parser kubernetes-audit
               DB /tail-db/audit.db
               Tag audit.*
-              Refresh_Interval 5
```

Comment:
What are the pros/cons for making the interval changes here?

Reply:
Refresh and Rotation Check Interval were even before (refresh 5 and default rotation wait 5).
In bigger clusters and long running we have seen that fluent-bit loses the tailing for auditlogs.
This change is possibly fixing it and has no real cons. Except it checks less often for changed Files to tail.